nfc/source/amiibo_char.h

```diff
@@ -0,0 +1,567 @@
+#ifndef _AMIIBO_CHAR_H

```

Comment:
Remove this file.

Reply:
Ok, was worried about possible copyright issues.

Reply:
Wait I misread...